tests/unit/modules/test_pyroute2.py

```diff
@@ -0,0 +1,12 @@
+
+from pyroute2 import IPDB
+from tests.support.unit import TestCase
+from salt.beacons.network_settings import ATTRS
+
+
+class Pyroute2TestCase(TestCase):
+
+    def test_interface_dict_fields(self):
+        with IPDB() as ipdb:
+            for attr in ATTRS:
+                self.assertIn(attr, ipdb.interfaces[1])
```

Comment:
Just to elaborate on the code, since I'm not sure I understand the comment correctly.

The issue #41461 was filed after an exception — there was `change` attr in `ATTRS`, removed from the `IPDB` high-level logic.

The condition on line 12 checks, if each and every attr in `ATTRS` is available via the `Interface` dict in `IPDB` (here `ipdb.interfaces[1]` in 99% will mean the loopback interface, I believe it's OK)

Thus we can be sure that `ATTRS` is a subset of the `pyroute2.ipdb.interfaces.Interface` API. 

Excuse me, if I miss something :)